.github/workflows/ci.yml

```diff
@@ -105,7 +105,7 @@ jobs:
       fail-fast: false
       matrix:
         os: [ubuntu-latest, macos-latest, windows-latest]
-        rust: [1.40.0]
+        rust: [1.42.0]
```

Comment:
In this case the workspace doesn't build unless the Rust version is bumped. Is there another way to handle it?

Thanks for the code review! I'll do the changes asap.

Reply:
Oh yes, that testcase runs on the workspace, I thought it would only run on the `sentry` crate…
I’m fine with just bumping MSRV, unless someone else from the team objects.